spring-boot-autoconfigure/src/test/java/org/springframework/boot/autoconfigure/cache/CacheAutoConfigurationTests.java

```diff
@@ -161,13 +173,13 @@ public void simpleCacheExplicit() {
 	@Test
 	public void simpleCacheWithCustomizers() {
 		testCustomizers(DefaultCacheAndCustomizersConfiguration.class, "simple",
-				"allCacheManagerCustomizer", "simpleCacheManagerCustomizer");
+			"allCacheManagerCustomizer", "simpleCacheManagerCustomizer");
```

Comment:
Could you please push force an update of your PR without those formatting changes please? (there are more below)